### PR TITLE
Some more reports octane

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
A few more octane fixups for reports

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/addon/components/report-builder.js [hunk 1]
```diff
@@ -53,43 +54,53 @@ export default Component.extend({
...
-      );
-    },
+  /**
+   * @action onToggleMetricFilter
+   * @param {Object} metric
+   */
+  @action
+  onToggleMetricFilter(metric) {
+    this._expandFilters(() =>
+      (this.request.having || []).find(having => get(having, 'metric.metric.name') === get(metric, 'name'))
```
Comment (jkusa) [inline packages/reports/addon/components/report-builder.js:84]:
```suggestion
      (this.request.having || []).find(having => having.metric.metric.name === metric.name')
```

File: packages/reports/addon/components/report-builder.js [hunk 2]
```diff
@@ -53,43 +54,53 @@ export default Component.extend({
...
+   * @param {Object} parameters
+   */
+  @action
+  onToggleParameterizedMetricFilter(metric, parameters) {
+    this._expandFilters(() =>
+      (this.request.having || []).find(
+        having =>
+          get(having, 'metric.canonicalName') ===
+          canonicalizeMetric({
+            metric: get(metric, 'name'),
```
Comment (jkusa) [inline packages/reports/addon/components/report-builder.js:100]:
```suggestion
            metric: metric.name,
```

File: packages/reports/addon/components/report-builder.js [hunk 3]
```diff
@@ -53,43 +54,53 @@ export default Component.extend({
...
+   * @action onToggleParameterizedMetricFilter
+   * @param {Object} metric
+   * @param {Object} parameters
+   */
+  @action
+  onToggleParameterizedMetricFilter(metric, parameters) {
+    this._expandFilters(() =>
+      (this.request.having || []).find(
+        having =>
+          get(having, 'metric.canonicalName') ===
```
Comment (mattgraham1995) [inline packages/reports/addon/components/report-builder.js:98]:
```suggestion
          having.metric.canonicalName ===
```

File: packages/reports/addon/templates/components/unauthorized-table.hbs [hunk 4]
```diff
@@ -1,9 +1,9 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#navi-info-message classNames="navi-report-invalid__info-message navi-report-invalid__unauthorized"}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<NaviInfoMessage class="navi-report-invalid__info-message navi-report-invalid__unauthorized">
```
Comment (daniellevinson) [inline packages/reports/addon/templates/components/unauthorized-table.hbs:2]: `<span><NaviIcon @icon="lock" /></span>` ⬇️

**Pull request merged**